README.md

```diff
@@ -1,28 +1,33 @@
 # Dss Flappers
 
-Implementations of MakerDao surplus auctions, triggered on `vow.flap`. The current featured Flapper is `FlapperUniV2`.
+Implementations of MakerDao surplus auctions, triggered on `vow.flap`.
 
-### FlapperUniV2
+### Splitter
```

Comment:
"the `vow` is expected to hold at least the swapped amount" =>
"the `vow` is expected to hold at least the consumed amount"